src/view/com/util/images/ImageLayoutGrid.tsx

```diff
@@ -93,13 +94,28 @@ function ImageLayoutGridInner(props: ImageLayoutGridInnerProps) {
 const IMAGE_GAP = 5
```

Comment:
Is the comment above still correct? Might be nice to update if we changed the strategy in any way. Ideally it should be clear what the hacky part is (and whether we can ever remove the hack).

Reply:
Yea, the issue is actually still valid. But adding something to clarify why we're avoiding the margin solution on native.